code/game/turfs/simulated/floor/fancy_floor.dm

```diff
@@ -160,6 +160,18 @@
 /turf/open/floor/wood/cold/broken/seven
 	icon_state = "wood-broken7"
 
+/turf/open/floor/wood/parquet
+	icon_state = "wood-parquet"
+	broken_states = list("wood-parquet-broken", "wood-parquet-broken2", "wood-parquet-broken3", "wood-parquet-broken4", "wood-parquet-broken5", "wood-parquet-broken6", "wood-parquet-broken7")
+
+/turf/open/floor/wood/tile
+	icon_state = "wood-tile"
+	broken_states = list("wood-tile-broken", "wood-tile-broken2", "wood-tile-broken3")
+
+/turf/open/floor/wood/large
+	icon_state = "wood-large"
+	broken_states = list("wood-large-broken", "wood-large-broken2", "wood-large-broken3")
```

Comment:
make sure to set floor_tile to the type of tile object used to place these, or when they're pulled up they're turn back into regular wood tiles